### PR TITLE
Bugfix/theme issues 4565

### LINKED ISSUE #4513: Host/Service count box border bottom
## Describe the bug
There is a small border under the host/service count box. Is this intentional? Because other menu items with a count box do not have this border.

Screenshot 2021-08-24 at 10 20 32

### LINKED ISSUE #4565: Theme issues
A collection of issues regarding the dark/light theme:

- [x] Navigation Problem badges for active items (Firefox)
- [x] Disable hover effect for active `.level-1` and `.level-2` nav-items

### Navigation Problem badges for active items (Firefox)
Screen Shot 2021-10-20 at 14 55 26

### Disable hover effect for active `.level-1` and `.level-2` nav-items
Screen Shot 2021-10-20 at 15 26 48

### PR DESCRIPTION
**Pull request by flourish86**
fixes #4565
fixes #4513

### CODE REVIEW

File: public/css/icinga/base.less [hunk 1]
```diff
@@ -140,10 +140,10 @@
...
     --low-sat-blue-dark: #c0cccd;
 
-    --gray: #7F7F7F;
-    --gray-semilight: #A9A9A9;
-    --gray-light: #C9C9C9;
-    --gray-lighter: #EEEEEE;
+    --gray: #819398;
+    --gray-semilight: #94a5a6;
+    --gray-light: #d0d3da;
+    --gray-lighter: #e8ecef;
```
[the remark this answers is not in the document]
Comment (flourish86) [reply public/css/icinga/base.less:146]: The lighter gray colors blend in a lot better. Remember the flickering contrast (“Flimmerkontrast”) we discussed regarding the login screen link color (“pink” on blueish). It’s the same phenomenon, although a little more subtile.

Look at the comparison. The tinted values are a lot smoother and easier to read for text and less distracting for background colors.

Desktop Copy

File: public/css/icinga/menu.less [hunk 2]
```diff
@@ -173,11 +172,10 @@
     .var(color, menu-2ndlvl-active-color);
     .var(background-color, menu-2ndlvl-active-bg-color);
 
-    &:hover, &:focus {
+    &:focus {
       &:first-of-type,
       &:first-of-type ~ a {
         .var(color, menu-2ndlvl-active-hover-color);
-        .var(background-color, menu-2ndlvl-active-hover-bg-color);
```
[the remark this answers is not in the document]
Comment (flourish86) [reply public/css/icinga/menu.less:180]: > Personally, I don't see a reason why the background shouldn't change as well when hovering as it does when focusing. So this and the change above is superfluous to me.

It’s not. With the hover effect the UI promises something to happen, but that isn't (at least nothing expected) when clicking the already active item. This feels like something isn’t right. This is this reason why you don’t find popular examples that do it this way.

[the remark this answers is not in the document]
Comment (nilmerg) [reply public/css/icinga/menu.less:180]: Okay, then not on hover, but on focus something has to change.

File: public/css/icinga/base.less (hunk 1 above)
[the remark this answers is not in the document]
Comment (nilmerg) [reply public/css/icinga/base.less:146]: Thanx

File: public/css/icinga/menu.less (hunk 2 above)
[the remark this answers is not in the document]
Comment (flourish86) [reply public/css/icinga/menu.less]: ✅ I reinstated the focus background change again with 5667924

**Review by nilmerg — APPROVED** [review]

**Pull request merged**